### PR TITLE
Day07 zig - SOLUTION # 100!

### PR DESCRIPTION
**Pull request by Avokadoen**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,15 +11,15 @@ Welcome to this community project, where we collaboratively solve the 2020 editi
...
 | ruby      |  **2** |  1   |     |     |  1   |      |      |     |     |     |     |     |     |     |     |     |     |     |     |     |     |     |     |     |     |     |
 | rust      |  **1** |  1   |     |     |      |      |      |     |     |     |     |     |     |     |     |     |     |     |     |     |     |     |     |     |     |     |
 | bash      |  **0** |      |     |     |      |      |      |     |     |     |     |     |     |     |     |     |     |     |     |     |     |     |     |     |     |     |
 | php       |  **0** |      |     |     |      |      |      |     |     |     |     |     |     |     |     |     |     |     |     |     |     |     |     |     |     |     |
-| **Total** | **99** |**12**|**8**|**9**|**11**|**10**|**10**|**6**|**5**|**4**|**4**|**3**|**3**|**2**|**3**|**3**|**1**|**1**|**1**|**1**|**1**|**1**|**0**|**0**|**0**|**0**|
+| **Total** | **99** |**12**|**8**|**9**|**11**|**10**|**10**|**7**|**5**|**4**|**4**|**3**|**3**|**2**|**3**|**3**|**1**|**1**|**1**|**1**|**1**|**1**|**0**|**0**|**0**|**0**|
```
Comment (Arxcis) [inline README.md]: # 100 !!! :100: :clap: 
```suggestion
| **Total** | **100** |**12**|**8**|**9**|**11**|**10**|**10**|**7**|**5**|**4**|**4**|**3**|**3**|**2**|**3**|**3**|**1**|**1**|**1**|**1**|**1**|**1**|**0**|**0**|**0**|**0**|
```

**Review by Arxcis — APPROVED** [review]
You just beat me to # 100 by a couple of minutes :laughing: Approved!

**Comment by Avokadoen** [comment]
> You just beat me to # 100 by a couple of minutes laughing Approved!

dank :sunglasses:

File: README.md [hunk 2]
```diff
@@ -19,7 +19,7 @@ Welcome to this community project, where we collaboratively solve the 2020 editi
 | rust      |  **1** |  1   |     |     |      |      |      |     |     |     |     |     |     |     |     |     |     |     |     |     |     |     |     |     |     |     |
 | bash      |  **0** |      |     |     |      |      |      |     |     |     |     |     |     |     |     |     |     |     |     |     |     |     |     |     |     |     |
 | php       |  **0** |      |     |     |      |      |      |     |     |     |     |     |     |     |     |     |     |     |     |     |     |     |     |     |     |     |
-| **Total** | **99** |**12**|**8**|**9**|**11**|**10**|**10**|**7**|**5**|**4**|**4**|**3**|**3**|**2**|**3**|**3**|**1**|**1**|**1**|**1**|**1**|**1**|**0**|**0**|**0**|**0**|
+| **Total** | **100** |**12**|**8**|**9**|**11**|**10**|**10**|**7**|**5**|**4**|**4**|**3**|**3**|**2**|**3**|**3**|**1**|**1**|**1**|**1**|**1**|**1**|**0**|**0**|**0**|**0**|
```
Comment (Arxcis) [inline README.md:22]: :eyes: :+1:

**Pull request merged**